generator-core/src/main/java/br/com/generator/core/GeneratorExecutor.java

```diff
@@ -35,7 +35,7 @@ public File addDependecies(GeneratorOptions options) throws IOException {
         File fileGenerated = new File(options.getDestination());
         FileUtils.writeStringToFile(fileGenerated, contentReplaced);
 
-        System.out.println("Add dependencies in ".concat(options.getDestination()));
+        System.out.println("Add dependencies in ".concat(options.getName()));
```

Comment:
Consider using string formatting instead of `concat` for better readability. For example: `System.out.println(String.format("Add dependencies in %s", options.getName()));` and `System.out.println(String.format("Add properties in %s", options.getName()));`